usb/esp_tinyusb/include/tinyusb.h

```diff
@@ -55,7 +55,7 @@ typedef struct {
  */
 esp_err_t tinyusb_driver_install(const tinyusb_config_t *config);
 
-// TODO esp_err_t tinyusb_driver_uninstall(void); (IDF-1474)
```

Comment:
I was not able to find IDF-1474. Not sure if it's still relevant.

Reply:
that is for our internal issue tracker, you don't need to worry about it now!